packages/dds/merge-tree/src/test/index.ts

```diff
@@ -126,4 +126,4 @@ export {
 	TrackingGroupCollection,
 	UnassignedSequenceNumber,
 	UniversalSequenceNumber,
-} from "..";
+} from "../index";
```

Comment:
Just curious: was this change mechanically necessary?

Reply:
Yes. And in fact, it really should be ../index.js, but it can't right now because typescript.